newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/PointCutConfiguration.java

```diff
@@ -88,10 +88,6 @@ public boolean isEnabled() {
 
         boolean groupExplicitlyDisabled = !pointCutGroupConfig.getProperty("enabled", true);
         boolean classExplicitlyDisabled = !pointCutConfig.getProperty("enabled", true);
-        if (groupExplicitlyDisabled && classExplicitlyEnabled) {
```

Comment:
This is always false because if `classExplicitlyEnabled` was `true`, the if on line 84 would eval to true and return